### DISCUSSION TITLE
社区例会，议题征集

### DISCUSSION THREAD

**Discussion by data-infra** [tencentmusic/cube-studio#75] [Q&A]
有想咨询或者讨论的，这里可以问一下。提前准备下问题。

**Comment by zhuyaguang** [comment]
希望讲解下整体的代码结构

**Comment by chengleqi** [comment]
希望分享个MNIST之类的全流程Quick Start demo

**Comment by HamaWhiteGG** [comment]
hi， 有以下三个建议: 
1. 希望讲解下cube-studio的整体架构
2. 目前一些生产案例的分享，最好能包含一些踩坑经验。
3. 下一步规划简单讲解下。
HamaWhite
------------------------------------------------------------------
发件人：chengleqi ***@***.***>
发送时间：2022年8月31日(星期三) 20:46
收件人：tencentmusic/cube-studio ***@***.***>
抄　送：Subscribed ***@***.***>
主　题：Re: [tencentmusic/cube-studio] 9月5号，下午3点，答疑解惑 预留问题 (Discussion #75)
希望分享个MNIST之类的全流程Quick Start demo

**Comment by zhuyaguang** [comment]
一些二次开发的讲解

**Comment by cyxnzb** [comment]
二次开发+1，以及一些核心功能的使用简介，帮助文档还是有些抽象，想看一些实战操作
